web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -1,40 +1,87 @@
 import axios from 'axios';
 import { defaultEndpointPrefix, getCookie } from 'utils';
-import jwtDecode from 'jwt-decode'
-import toast from 'react-hot-toast';
 
-axios.defaults.headers.post['Content-Type'] = 'application/json';
-axios.defaults.baseURL = defaultEndpointPrefix();
+const instance = axios.create({
```

Comment:
Creating the instance and using the interceptors like this seems a lot simpler and the code is much reader. Very well written!